LICENSE

```diff
@@ -631,8 +631,8 @@ to attach them to the start of each source file to most effectively
 state the exclusion of warranty; and each file should have at least
 the "copyright" line and a pointer to where the full notice is found.
 
-    <one line to give the program's name and a brief idea of what it does.>
-    Copyright (C) <year>  <name of author>
+    <Woodog is an initiative to provide help to increasing stray animals by providing them food and refuge by connecting them with fellow helpful humans.>
+    Copyright (C) <2021>  <Feminine-Divine>
```

Comment:
Hi @thegautamkumarjaiswal, Just 2021 is enough, It doesn't need to be enclosed with angular brackets. Same for all, 
Please update this too. :+1:

Reply:
Ok, I do.